OpenRA.Mods.Common/Traits/Repairable.cs

```diff
@@ -91,6 +91,8 @@ public void ResolveOrder(Actor self, Order order)
 				var target = Target.FromOrder(self.World, order);
 				self.SetTargetLine(target, Color.Green);
 
+				TryCallTransport(self, target, movement);
+
 				self.CancelActivity();
```

Comment:
You call TryCallTransport just above, which queues its own set of repair activities... then cancel them and queue another set?